networkx/utils/tests/test_misc.py

```diff
@@ -239,9 +239,11 @@ def test_create_py_random_state():
     np = pytest.importorskip("numpy")
 
     rs = np.random.RandomState
+    rng = np.random.default_rng(1000)
```

Comment:
Would it be worth also adding an explicit test of `np.random.Generator` (which I assume is what the `default_rng` returns...  but that could change in the future!)

Reply:
AIUI `default_rng` will always return a `Generator` instance, though there is not necessarily a guarantee that it will have the same `BitGenerator`... either way, it's simple enough (and an improvement) to be explicit!